invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/theme.js

```diff
@@ -28,7 +28,7 @@ $(".dropdown.export").dropdown({
   },
 });
 
-const $licensesPopup = $("#licenses li.has-popup .license.clickable");
+const $licensesPopup = $("#licenses dd.has-popup .license.clickable");
```

Comment:
looks more sleek, I like it!

for js and css changes, please remember to keep backwards compatibility for these elements, for anyone who is overriding the templates and still uses `li`, otherwise it will break for them - same comment for the .less changes

Reply:
you can include the old class as one of the element selectors